docs/source/markdown/options/network.md

```diff
@@ -30,3 +30,43 @@ Valid _mode_ values are:
   - **port_handler=rootlesskit**: Use rootlesskit for port forwarding. Default.
   Note: Rootlesskit changes the source IP address of incoming packets to an IP address in the container network namespace, usually `10.0.2.100`. If your application requires the real source IP address, e.g. web server logs, use the slirp4netns port handler. The rootlesskit port handler is also used for rootless containers when connected to user-defined networks.
   - **port_handler=slirp4netns**: Use the slirp4netns port forwarding, it is slower than rootlesskit but preserves the correct source IP address. This port handler cannot be used for user-defined networks.
+
+
+- **pasta[:OPTIONS,...]**: use **pasta**(1) to create a user-mode networking
```

Comment:
If the intention is to use `pasta` as an arg to `--network`, then you need to add a one-liner on or about line 12.

Reply:
I think what is described around line 12 are the options for the `bridge` `mode`, right? Am I missing something? I read and re-read and I can't understand why I should add something there. Yes, the intention is to use `pasta` as a `mode` argument for `--network`.

Reply:
Yes, you're right. I misread the option description - sorry about that.

Reply:
It might be me, but I'm not finding the pasta man page.  We generally only add a man page link to the bottom of the page, which we should do here.   But if the pasta man page is as hard to find for others, I think keeping the link here in the opening line is good, but please remove them elsewhere.

Reply:
In which way is it hard to find? If you're using a package, official or unofficial, (see the "Availability" section of the pull request message), `man pasta` should be enough. If you're building from source, it's also copied by `make install`. In both cases, it's generated here:
    https://passt.top/passt/tree/Makefile?id=ea5e046646638307cc4404c7c16a4a0b54644fba#n133

please let me know if you're hitting any specific issue, so that I can fix it.